man/roxygen/meta.R

```diff
@@ -16,7 +16,6 @@ list(
     utils_fmt = "Utilities for Formatting Observations: ",
     utils_help = "Utilities used within Derivation functions: ",
     utils_examples = "Utilities used for examples and template scripts: ",
-    utils_impute = "Utilities used for date imputation: ",
```

Comment:
Umm....this is for the families right?  Should we keep this if we are keeping the @families in the roxygen?

Reply:
`utils_impute` was the keyword that was used to group these functions in the Reference page, which we no longer are using. All these non-exported functions still have the `@family utils_impute` tag, which links the help files together in the See Also section, e.g.

<img width="566" alt="image" src="https://github.com/pharmaverse/admiral/assets/26774684/ce3c9eec-92c8-4b0f-a6fe-7d8e59cd3f37">

To be 100% upfront, I've never used the `meta.R` roxygen2 script. Are you seeing anything unexpected in the documentation?

Reply:
Did more reading, and you're 100% correct about `meta.R`. Here's what it looks like with that entry back: (the family title is modified)

<img width="815" alt="image" src="https://github.com/pharmaverse/admiral/assets/26774684/bd0e351b-31c1-475d-a30b-1712109e965b">

Reply:
Awesome.  Yeah I set this up, but couldn't remember what our decision was around families for functions not visible on website.  I think this is nice for a developer when looking for other dtm imputation functions.